### PR TITLE
Added documentation for deployment, migration, etc...

### PR DESCRIPTION
**Pull request by sasbury**
Fixed some markdown warnings
Added revocation doc
Added a bit about URLs to nsc doc
Updated nsc overview to discuss the env more.
Added info on defining your own well known operator.

### CODE REVIEW

File: nats_tools/nsc/managed.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Working with Managed Operators
+
+`nsc` can be used to administrate multiple operators. Operators can be thought of as the owners of nats-servers, and fall into two categories: local and managed. The key difference, pardon the pun, is that managed operators are ones which you don't have the nkey for. An example of a managed operator is the Synadia service called NGS. Synadia has the keys.
```
Comment (gcolliso) [inline nats_tools/nsc/managed.md]: could we swap administer for administrate?

File: nats_tools/nsc/managed.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Accounts, as represented by their JWTs, are signed by the operator. Some operators may use local copies of JWTs, others may use the [nats-account-server](../nas/README.md) to manage their JWTS. Synadia uses a custom server for their JWTs that works similarly to the open-sourced account server.
```
Comment (gcolliso) [inline nats_tools/nsc/managed.md]: JWTS ->JWTs

File: nats_tools/nsc/nsc.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-## NSC
+# NSC
 
 NATS uses JWTs to armor the various identity and authorization artifacts. These JWTs are created with the `nsc` tool. NSC simplifies the tasks of creating and managing  identities and other JWT artifacts.
```
Comment (gcolliso) [inline nats_tools/nsc/nsc.md]: spacing issues - remove first space from the paragraph and between 'managing' and  'identities'

**Comment by sasbury** [comment]
Fixed all the comments - I think.

File: nats_tools/nsc/nsc.md [hunk 4]
```diff
@@ -20,11 +20,11 @@ In this guide, you’ll run end-to-end on some of the configuration scenarios:
 
 Let’s run through the process of creating some identities and JWTs and work through the process.
 
-By default JWTs are written to `~/.nsc` and secrets to `~/.nkeys`. You can easily change those locations by setting `NSC_HOME` and `NKEYS_PATH` respectively in your environment to your desired locations.
+## The NSC Environment
 
-> The $NKEYS_PATH stores secrets. Since nkeys relies on cryptographic signatures to prove identity, anyone with access to your private keys will be able to assume your identity. With that said, treat them as secrets and guard them carefully.
+By default JWTs are written to `~/.nsc` and secrets to `~/.nkeys`. `nsc` also tracks a value called the "stores directory". This directory is the one that contains the operators you are currently working with. By default the stores directory is `~/.nsc/nats` but you can switch it to another folder if you want to separate JWTs for use in a revision control system, or co-locate them with a project, etc...
```
Comment (gcolliso) [inline nats_tools/nsc/nsc.md]: By default JWTs are written to `~/.nsc` and secrets to `~/.nkeys`. `nsc` also tracks a value called the "stores directory". This directory contains the operators you are currently working with. By default the stores directory is `~/.nsc/nats` but you can switch it to another folder if you want to separate JWTs for use in a revision control system, or co-locate them with a project, etc..

Comment (sasbury) [reply nats_tools/nsc/nsc.md]: fixed in latest commit

File: nats_tools/nsc/revocation.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+Available Commands:
+  add_activation    Revoke an accounts access to an export
+  add_user          Revoke a user
+  delete_activation Remove an account revocation from an export
+  delete_user       Remove a user revocation
+  list_activations  List account revocations for an export
+  list_users        List users revoked in an account
+```
+
+both add commands take the flag `--at` which defaults to 0, for now, which can be used to set the unix timestamp as described above. By default revocations are at the current time, but you can set them in the past for situations where you know when a problem occurred and was fixed.
```
Comment (gcolliso) [inline nats_tools/nsc/revocation.md]: Both

Comment (sasbury) [reply nats_tools/nsc/revocation.md]: pushed a fix

**Comment by gcolliso** [comment]
lgtm - thanks @sasbury

**Pull request merged**